### PR TITLE
Data panel fixes

### LINKED ISSUE #7939: Data panel improvements and fixes
- [x] Show a loading while filling the blank.
- [x] Avoid requests if the schema model has not changed when we are re-drawing the stats.
- [x] Show the style link (and connect it) only for added widgets.

### PR DESCRIPTION
**Pull request by nobuti**
This PR fixes #7939.

When creating stat views, we only created an instance of Tablestats in order to take advantage of a persisted object in order to make queries to the sql api. This way only one request is made for the stats. Also, it adds the preload view and connect the style button to the proper view.

cc @xavijam

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/data/data-content-view.js [hunk 1]
```diff
@@ -12,35 +13,53 @@ module.exports = CoreView.extend({
...
+      dataNotReadyTemplate()
+    );
+  },
+
   _initBinds: function () {
     this._querySchemaModel.on('change', this._onQuerySchemaChange, this);
     this.add_related_model(this._querySchemaModel);
-    this._querySchemaModel.fetch();
-
     this._optionsCollection.on('change:selected', this._handleWidget, this);
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/data/data-content-view.js]: `this.add_related_model(this._optionsCollection);`

**Comment by xavijam** [comment]
Could you add an image? :)

**Comment by xavijam** [comment]
:ship: :it:

**Comment by nobuti** [comment]
data-fixes

**Pull request merged**